tools/cli/commands/changelog.js

```diff
@@ -834,44 +823,56 @@ async function promptChangelog( argv, needChangelog ) {
  */
 async function changelogAddPrompt( argv, needChangelog, uniqueProjects ) {
```

Comment:
The `needChangelog` argument should be renamed to `defaultProjects` to better reflect what it is (and the doc comment above should be updated accordingly).

Reply:
Good thinking 👍 48567a9057bfb7a292ca9664980130d9c11fc379